numbat-wasm/www/editor.css

```diff
@@ -5,33 +5,108 @@ html {
     margin: 0;
     padding: 0;
     height: 100%;
+
+    /* Maybe lower font-size to 16px? */
```

Comment:
I'm okay with a slightly smaller font size than 30px but then we probably want to move the code and the results closer together somehow? (e.g. padding on both sides of the screen)

Reply:
For now I have set it to 18px while I'm working on it, but I will look into how to make the code and results work with the smaller font later. I will try to make the design responsive to screen size so it will look decent on anything from a phone to a wide display, but it will take some fiddling around for me to get there.